### ISSUE TITLE
xLights is not using port 0x1936 (6454) as the source port when transmitting ArtNet data.  

### ISSUE THREAD

**Issue by JanWull** [smeighan/xLights#3292]
**Describe the bug**
xLights will not output to a particular ArtNet node.  The node shows up in the controller list, the node still works fine with other test software, and xLights works fine with the other two nodes (different manufacturers).  After analyzing the network data I've noticed that xLights does NOT use port 6454 as the source port for ArtNet, and uses arbitrary port numbers.  This seems to be the only difference in the network data compared to other software.  Reading up on the ArtNet specs it says that port 6454 (0x1936) should be used.  I have a feeling that this one node in particular will only communicate on that port, while the other two nodes are a little more lax about source port #'s.  

**To Reproduce**
1. Install a Showtec Node 8 on the network
2. Set up the controller in xLights
3. Create a basic test sequence
4. Click "Output To Lights"
5. Observe network data in DMX-Workshop->Developer->Network packet dissector

**Expected behavior**
Source port to be 0x1936 as per the ArtNet spec.

**Screenshots**
![artnet2](https://user-images.githubusercontent.com/72585621/175354046-47996324-2654-4ac2-800f-4c9838e2053c.jpg)
![artnet3](https://user-images.githubusercontent.com/72585621/175354575-02e5f1be-6967-454f-b428-6a03e32c5990.jpg)

**Versions (please complete the following information):**
 - Windows 10
 - xLights version 2022.12

**Comment by JanWull** [comment]
Issue has been found and pull request created. :)

**Comment by keithsw1111** [comment]
I do not believe this is an xLights defect. With protocols like this the receiver uses the well known port number. The sender can use any port at all. If the sender had to use the well known port then a machine sending artnet could not receive artnet as well and only one piece of software on a machine could send artnet. There is no reason I can think of why you would do that. I think you are misreading the spec as it is only the receiver that must listen on 0x1936.
